test/e2e/helpers/visu/PageTester.ts

```diff
@@ -16,18 +16,11 @@
 import { FitType, LoadOptions } from '../../../../src/component/options';
 import { ElementHandle } from 'puppeteer';
 import { BpmnQuerySelectors } from '../../../../src/component/registry/bpmn-elements-registry';
-import { copyFileSync, loadBpmnContentForUrlQueryParam } from '../../../helpers/file-helper';
-import { log } from '../test-utils';
-
-export enum BpmnLoadMethod {
```

Comment:
⚠️ removing load method also means updating the documentation
See 'BPMN diagram load method' in contributors/testing.md

Reply:
Doc updated